Packages/com.unity.inputsystem/InputSystem/InputUpdateType.cs

```diff
@@ -71,27 +71,95 @@ public enum InputUpdateType
 
     internal static class InputUpdate
     {
-        public static bool s_HaveUpdatedStepCount;
         public static InputUpdateType s_LastUpdateType;
-        public static uint s_UpdateStepCount;
+        public static UpdateStepCount s_PlayerUpdateStepCount;
+        #if UNITY_EDITOR
+        public static UpdateStepCount s_EditorUpdateStepCount;
+        public static uint s_UpdateStepCount => s_LastUpdateType == InputUpdateType.Editor ? s_EditorUpdateStepCount.value : s_PlayerUpdateStepCount.value;
+        #else
+        public static uint s_UpdateStepCount => s_PlayerUpdateStepCount.value;
+        #endif
         public static uint s_LastUpdateRetainedEventBytes;
         public static uint s_LastUpdateRetainedEventCount;
 
+        [Serializable]
+        public struct UpdateStepCount
+        {
+            public bool wasUpdated;
```

Comment:
maybe make these fields private and introduce properties with only getters,  since as far as I understood, it's not intended to change wasUpdated/value outside of this struct?

Reply:
actually wasUpdated  is not being used outside, so no need to add property for it

Reply:
Good idea! Will do

Reply:
Fixed